scripts/avocado_featurize_submit

```diff
@@ -57,16 +76,21 @@ if __name__ == "__main__":
         default=None,
         help='Working directory. Default is the current directory.'
     )
+    parser.add_argument(
+        '--schedular',
```

Comment:
Is there a reason why you chose "schedular" for this instead of "scheduler"? I think that I would prefer "scheduler".

Reply:
Ah, that's interesting! I had never realised it should be spelt that way and my spellchecker just never caught it as well. I am happy to change to "scheduler", no problem.

Resolved with [`aa09b49`](https://github.com/kboone/avocado/pull/6/commits/aa09b49cfef3db4802d97a2c2ded51155f5057f6)